mlflow/sagemaker/__init__.py

```diff
@@ -1064,6 +1066,33 @@ def _sigterm_handler(*_):
     proc.wait()
 
 
+def target_help(_):
+    """
+    Provide help information for the SageMaker deployment client.
+
+    :return:
+    :rtype: str
+    """
+    help_str = """For detailed documentation on the SageMaker deployment client, please visit
+    https://mlflow.org/docs/latest/python_api/mlflow.sagemaker.html#mlflow.sagemaker.SageMakerDeploymentClient
+
+    The target URI must follow the following formats:
+    - sagemaker
+    - sagemaker:/region_name
+    - sagemaker:/region_name/assume_role_arn
+
+    When the region_name or assume_role_arn are provided, they will be used as the default region
+    and assumed role ARN when executing the commands.
+
+    The `create` and `update` commands require a deployment name and a model_uri. The model flavor
+    and deployment configuration can be optionally provided.
```

Comment:
Nice docstring! Can we also clarify that sync & async execution are supported, since we highlight that for `delete` below as well?